packages/malloy/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@malloydata/malloy",
-  "version": "0.0.43",
+  "version": "0.0.44",
```

Comment:
@whscullin I've been bumping this incrementally (so that my `malloy-link` version doesn't conflict with the current real version), but I don't think I'm actually doing the right thing here. Do we have a strategy for keeping this up-to-date?

Reply:
You shouldn't need to to touch that to use link.